File: hw/sisalema/u1/hw09SaveAnObject_(Phone)/Phone_Javascipt/ec/edu/espe/saveanobject/view/ClockSystem.js
import { Serializable } from "utils./Serializable.js";
import readlineSync from 'readline-sync';

class ClockSystem {
    static main(){          
        let clocks = [];
        let continueInput = true;
        do {
            console.log("1. Create clock");
            console.log("2. Read Clocks");
            console.log("3. Exit");
            let option = readlineSync.question("Choose an option:");
        
            switch (option) {
                case "1":
                    Serializable.createClock(clocks);
                    break;
                case "2":
                    Serializable.read(clocks);
                    break;
                case "3":
                    continueInput = false;
                    break;
                default:
                    console.log("Invalid option. Please try again.");
            }
        } while (continueInput);
    }
}
ClockSystem.main();

File: hw/sisalema/u1/hw09SaveAnObject_(Phone)/Phone/src/ec/edu/espe/saveanobject/view/ClockSystem.java
package ec.edu.espe.saveanobject.view;

import ec.edu.espe.saveanobject.model.Clock;
import ec.edu.espe.saveanobject.utils.Serializable;
import java.util.Scanner;
import java.util.ArrayList;

/**
 *
 * @author Yostin Sisalema,Code Master,DCC0-ESPE
 */

 public class ClockSystem {
        public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        ArrayList<Clock> clocks = new ArrayList<>();
        boolean continueInput = true;
        do {
            System.out.println("1. Create clock");
            System.out.println("2. Read Computer");    
            System.out.println("2. Exit");
            System.out.print("Choose an option: ");
            int option = scanner.nextInt();
            scanner.nextLine();

            switch (option) {
                case 1:
                    Serializable.createCock(scanner,clocks);
                    break;
                case 2:
                    Serializable.read(scanner,clocks);
                    break;
                case 3:
                    continueInput = false;
                    break;
                default:
                    System.out.println("Invalid option. Please try again.");
            }
        } while (continueInput);

        Serializable.saveJson(clocks);
    }
}